web_ckeditor4/static/src/js/web_ckeditor4.js

```diff
@@ -214,11 +220,18 @@ openerp.web_ckeditor4 = function(instance)
             this._cleanup_editor();
         }
     });
-    instance.web_ckeditor4.FieldCKEditor4Raw = instance.web_ckeditor4.FieldCKEditor4.extend({
+    var FieldCKEditor4Raw = FieldCKEditor4.extend({
         filter_html: function(value)
         {
             return value;
         }
     });
-}
-
+    core.form_widget_registry.add('text_ckeditor4', FieldCKEditor4);
+    core.form_widget_registry.add('text_ckeditor4_raw', FieldCKEditor4Raw);
+    core.form_widget_registry.add('text_html', FieldCKEditor4);
+    core.form_widget_registry.add('html', FieldCKEditor4);
```

Comment:
Why overriding html default widget? Would it be better to only use text_ckeditor4 and text_ckeditor4_raw widgets so that default html widget is still working? Or maybe add another widget `default_html` that allows someone to keep the default editor where needed.